manifests/init.pp

```diff
@@ -104,6 +104,8 @@
 #
 # $https_chain::                apache chain file for ssl
 #
+# $https_ca_cert::              apache ca cert file for ssl, falls back to ca_cert if empty
```

Comment:
ntipick s/empty/undefined/ 
Obviously depending on how you feel about my comment below.